### PR TITLE
Correction to newsletter Validation 

### PR DESCRIPTION
**Pull request by Samju200**
## Task: `Newsletter validation`
- `Changes you made to given task`

### CODE REVIEW

**Review by adaugochi — APPROVED** [review]

File: submissions/adeyemiJuwon/dom/newsletter.js [hunk 1]
```diff
@@ -1,35 +1,22 @@
...
-    }
-    }
-    
-    function emailFieldValidation(event){
-        emailRegex = /^\w+[\w-\.]*\@\w+((-\w+)|(\w*))\.[a-z]{2,3}$/;
-        if (!email.value.match(emailRegex)){
+    function validateForm(e){
+        if(!email.value.match(emailRegex)){
             document.querySelector(".error").innerHTML = "Please enter a valid email";
             event.preventDefault();
```
Comment (adaugochi) [inline submissions/adeyemiJuwon/dom/newsletter.js:9]: This should be `e` not `event`. Except you want to pass `event` as the parameter

File: submissions/adeyemiJuwon/dom/newsletter.js [hunk 2]
```diff
@@ -1,35 +1,22 @@
...
             return false;
-        } else{
-            document.querySelector(".error").innerHTML =  "";
+        }
+         else if(email.value === ""){
+            document.querySelector(".error").innerHTML = "Please enter your Email Address";
+            event.preventDefault();
```
Comment (adaugochi) [inline submissions/adeyemiJuwon/dom/newsletter.js:14]: same here

**Review by adaugochi — COMMENTED** [review]
I guess you just use it but don't really know why

File: submissions/adeyemiJuwon/dom/newsletter.js (hunk 1 above)
Comment (Samju200) [reply submissions/adeyemiJuwon/dom/newsletter.js:9]: Yes I thought both can work together. either e or event should work the same way.

**Review by ToluAkin — CHANGES_REQUESTED** [review]
@Samju200 I mentioned removing the class `.error` from the submit buttons of product and contact page.

**Comment by Samju200** [comment]
> @Samju200 I mentioned removing the class `.error` from the submit buttons of product and contact page.

@ToluAkin we are not suppose to validate the product and contact page newsletter

**Comment by ToluAkin** [comment]
> > @Samju200 I mentioned removing the class `.error` from the submit buttons of product and contact page.
> 
> @ToluAkin we are not suppose to validate the product and contact page newsletter

Oh! Yeah. That's fine.

**Review by ToluAkin — APPROVED** [review]

**Pull request merged**